packages/dashmate/package.json

```diff
@@ -103,6 +103,9 @@
     "sinon": "^11.1.2",
     "sinon-chai": "^3.7.0"
   },
+  "resolutions": {
```

Comment:
I think `resolutions` works only in root package.json

Reply:
But when we build dashmate, this files becomes our root package

Reply:
Shall we remove it then from the root one?